vignettes/prompt-design.Rmd

```diff
@@ -226,7 +226,7 @@ chat <- chat_openai(instruct_json)
 chat$chat(ingredients)
 ```
 
-(I don't know if the colour text, "You're an expert baker who also loves JSON", does anything, but I like to think this helps the LLM get into the right mindset of a very nerdy baker.)
+(I don't know if the text colouring, "You're an expert baker who also loves JSON", does anything, but I like to think this helps the LLM get into the right mindset of a very nerdy baker.)
```

Comment:
I think this changes the meaning, which suggests "colour text" isn't clear. Maybe something like "I don't know if the hint ...` would be better?

Reply:
(I don't know if the hint or colour text, "You're an expert baker who also loves JSON", actually does anything, but I like to think it helps the LLM get into the mindset of a very nerdy baker.)